bin/tfr-read

```diff
@@ -40,19 +40,36 @@ def get_args():
     return parser.parse_args()
 
 
-def list_tf_records(paths, schema):
+def resolve_dir_schemas(dirs):
+    dir_schemas = []
+    for d in dirs:
+        s = os.path.join(d, "_schema.pb")
+        s_inferred = os.path.join(d, "_inferred_schema.pb")
+        if file_io.file_exists(s):
+            dir_schemas.append((d, s))
+        elif file_io.file_exists(s_inferred):
+            dir_schemas.append((d, s_inferred))
+        else:
+            dir_schemas.append((d, None))
+    return dict(dir_schemas)
+
+
+def list_tf_records(paths, default_schema):
```

Comment:
What is this `default_schema` used for?

Reply:
it's the schema that's passed by the user. 
I changed the behavior: if passed explicitly, then it's not overridden